.github/workflows/main.yml

```diff
@@ -10,7 +10,10 @@ on:
 
 jobs:
   jekyll:
-    runs-on: ubuntu-16.04 # can change this to ubuntu-latest if you prefer
+    permissions:
+      deployments: write
```

Comment:
Referencing: https://docs.github.com/en/rest/reference/permissions-required-for-github-apps#permission-on-deployments